cityofaustin/atd-moped
# Fixes component form so that signal autocomplete input renders

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/12810

Well, this one slipped through release testing :/ It stems from a minor change that Mike made at the end of the PR review process that I should have caught in testing. Ivonne brought it to my attention so I think we'll want to bring this forward as a patch release as well. 


## Testing
Local or Netlify (https://deploy-preview-1074--atd-moped-main.netlify.app/moped/projects)

**Steps to test:**
1. Create a new component. Select the  "Signal - Traffic" or "Signal - PHB" component
2. Observe that the autocomplete input appears and enables you to search/select a traffic signal or PHB.
3. Change the component type to something else. The autocomplete input goes away.
4. Save your signal component. Edit it. Observe that the autocomplete input is visible in the form.


---
#### Ship list
